src/index.ts

```diff
@@ -57,41 +88,46 @@ module.exports = (app) => {
     const github = await app.auth(12536268); // Insert Installation ID here
     const owner = issue.owner;
     const repo = issue.repo;
-    const state = issue.state;
     return await github.issues.update({
       owner: owner,
       repo: repo,
       issue_number: issueId,
-      state: state,
+      state: "closed",
     });
 
   }
 
-  router.post('/Issue', (req,res) => {
+  // To create a new issue
+  router.post('/Issue', async (req,res) => {
     const password = req.headers.authentication;
     if(password === process.env.PASSWORD){
-      // Send a request to github through octokit
-      console.log(req.body.title)
+      // Do body preprocessing here
+      let actualBody = 'By: @' + req.body.author + '\n\n**BODY**\n' + req.body.Body + '\n\n**DIFFICULTY**\n';
+      actualBody += (parseInt(req.body.Difficulty) == 0) ? 'Beginner' : (parseInt(req.body.Difficulty) == 1) ? 'Intermediate' : 'Advanced';
+      actualBody += '\n### Note\n The time and venue will be decided in the Telegram group, so make sure you\'ve joined up the group.'
+      console.log(actualBody);
       const issue = {
         owner: 'Vikhyath08',
```

Comment:
And i guess having the repo name as env varibale would be better and the auther is supposed to be sent from the telegram bot 
**Pretty sure you know all this but structurizing** and try to look professional :sunglasses: